### ISSUE TITLE
MLM commission conditions

### ISSUE THREAD

**Issue by palepaul** [anujtenani/goaffpro#125]
Quite a few merchant have messaged recently saying that they would like an option where their affiliates only get their MLM commission (credited to their account) when they reach a particular sales figure themselves (from referrals). 

Merchant have messaged that they don't like that some of their parent affiliates are now entirely earning money from network commissions and not doing any referral sales of their own.

**Comment by hadstyle** [comment]
Thank you raising this.

This would be beneficial to my business if you could bring this in.

**Comment by KalaiaProducts** [comment]
This is very standard in the MLM world. Usually you have certain personal activity requirements to earn each tier of commissions. We would love to see this implemented ASAP so we can open up level commissions for our affiliates. This is the only reason why we haven't.

**Comment by ProAthletesHemp** [comment]
I agree.

**Comment by matwave123** [comment]
Please,please implement this, I am sure most businesses need it!

**Comment by vitasara** [comment]
yes, i agree. please implement this as soon as possible. Because my affiliates business need that urgently. now I do this manually. I hope I can also revise the multi-level commissions manually also, but now it is fixed, can't be changed.

**Comment by matwave123** [comment]
What do you do manually? 🙏👍

**Comment by SuperNovaCreations** [comment]
Yes this feature is really needed in the MLM world.

**Comment by matwave123** [comment]
Are we getting it please 


Appointments are only booked once you have had a confirmation email!

Matthew Brownell L.C.S.P. (Assoc)
Health & well being specialist of the year 2019

Stress solutions sessions Ltd
38 King street west
Manchester 
M3 2WZ


www.Matthewbrownell.co.uk 
www.Egglixir.com

**Comment by palepaul** [comment]
This option is available (Compression): https://docs.goaffpro.com/affiliate-commissions/create-commission-groups-for-affiliates/configure-compression-in-multi-level-network-for-group

How to use it: 
i. You can create an affiliate group (enable from Settings > Extras section and go the Groups tab to create the group)
ii. Add affiliates to the group >  set up the MLM rates within the group > finally set up compression for the MLM structure

Thank you
